src/js/components/Ballot/BallotTitleHeader.jsx

```diff
@@ -109,6 +111,33 @@ class BallotTitleHeader extends Component {
       const showEditAddress = true;
       const showSelectBallotModal = true;
       // this.props.toggleSelectBallotModal('', showEditAddress, false);
+      const { location: { pathname: currentPathname } } = window;
+      const page = lookupPageNameAndPageTypeDict(currentPathname);
+      const dataLayerObject = {
+        actionDetails: {
+          actionType: 'openModal',
+          buttonId: 'editAddressButton',
```

Comment:
Since onClick={this.showSelectBallotModalEditAddress} is called from a few different places, please pass the buttonId into showSelectBallotModalEditAddress so it is a variable instead of being hardcoded as 'editAddressButton'. Like this where 'ballotTitleBallotAddress' matches the id of the element where the onClick event is:
```
onClick={() => this.showSelectBallotModalEditAddress('ballotTitleBallotAddress')}
```